### PR TITLE
Add missing CPU variant for all ARM images

### PR DESCRIPTION
**Pull request by lbcjbb**
Add missing CPU variant for all ARM images.

Closes: #1382

### CODE REVIEW

File: base/base.bzl [hunk 1]
```diff
@@ -103,6 +103,7 @@ def distro_components(distro):
                 workdir = workdir,
                 os = "linux",
                 architecture = arch,
+                variant = VARIANTS.get(arch),
```
Comment (loosebazooka) [inline base/base.bzl:106]: presumably this just bubbles down to all images?

Comment (lbcjbb) [reply base/base.bzl:106]: Yes

**Comment by loosebazooka** [comment]
Anything in particular you need this for, just out of curiousity? Was this blocking some process for you?

**Review by loosebazooka — APPROVED** [review]

**Comment by lbcjbb** [comment]
No, it's not actually a barrier for me. This issue is trivial, but in the future with ARM64v9 (future AWS Graviton 4?) the CPU variant must be required.

**Pull request merged**